app/bundles/CoreBundle/Views/Helper/page_actions.html.php

```diff
@@ -19,9 +19,9 @@
 }
 
 include 'action_button_helper.php';
-
-echo '<div class="std-toolbar btn-group">';
-
+if (!empty($templateButtons)) {
```

Comment:
Is there a reason this is only getting applied to $templateButtons now when there may be custom buttons in the toolbar?